### PR TITLE
Add Type.paramLists

### PR DESCRIPTION
**Pull request by dwijnand**

### CODE REVIEW

File: core/src/main/scala_2.10/macrocompat/macrocompat.scala [hunk 1]
```diff
@@ -200,6 +200,8 @@ trait MacroCompat {
     def dealias: Type = tpe.normalize
 
     def finalResultType: Type = (tpe: global.Type).finalResultType
+
+    def paramLists: List[List[global.Symbol]] = tpe.paramss
```
Comment (dwijnand) [inline core/src/main/scala_2.10/macrocompat/macrocompat.scala]: This is actually problematic..

Comment (milessabin) [inline core/src/main/scala_2.10/macrocompat/macrocompat.scala]: Shouldn't it be `List[List[Symbol]]`?

Comment (dwijnand) [inline core/src/main/scala_2.10/macrocompat/macrocompat.scala]: Yeah.. seems `paramss` is only on `scala.tools.nsc.Global.Type`.. need to cast it back.

Updated the pull request, what do you think?

**Comment by milessabin** [comment]
LGTM ... merging :-)

**Pull request merged**